config/env.ts

```diff
@@ -23,11 +25,16 @@ const services = {
   }
 };
 
+const twilio: TwilioConfig = {
+  sid: 'AC24a1e734285a8c2ca2c1efc8ed86cdbc',
```

Comment:
Just move this to env variables. You can add these on server in `docker-compose.yml` adding keys in open source projects is dangerous.